storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,126 @@
+.. _page_tracking:
+
+====================================
+Page Tracking for Incremental Backup 
+====================================
+
+This feature is *tech preview* quality.
+
+Percona XtraBackup 8.0.27 adds support for the page tracking functionality for `incremental backup <https://www.percona.com/doc/percona-xtrabackup/8.0/backup_scenarios/incremental_backup.html>`__ . 
+
+To create an incremental backup with page tracking, Percona XtraBackup (PXB) uses the MySQL mysqlbackup component. This component looks in the InnoDB data files for pages that have been modified since the last backup and copies only those changed pages. This operation removes the need to scan all of the pages in the database. If the number of changed pages is not over 50%, the page tracking feature can improve the speed of incremental backups.
```

Comment:
This component provide the list of modified pages since the last backup so PXB can copy only those pages and avoid the the full scan of all the pages in the database.

Remove this number. we can place this at later section. This is not a concrete number 50%

Reply:
Does the component provide a list? According to other documentation, the list of changed pages is stored in the datadir and is not stored in the component.

Reply:
updated